internal/cmd/controller/imagescan/tagscan_job.go

```diff
@@ -64,24 +64,22 @@ func NewTagScanJob(c client.Client, namespace string, name string) *TagScanJob {
 	}
 }
 
-func (j *TagScanJob) Execute(ctx context.Context) {
+func (j *TagScanJob) Execute(ctx context.Context) error {
```

Comment:
We need to add that return value because the new [quartz.Job](https://github.com/reugn/go-quartz/blob/v0.11.1/quartz/job.go) needs it.

We're already doing our own error handling with the `updateErrorStatus` functions.

`quartz` retries the command in case of error, if `MaxRetries` is set in the job. (That's the use of the error returned, note that this `Execute` is called from the internal library loop, so it's not something we can access)

We could let the library do retries, but this is just bumping the version.... I'm not sure we want to do retries from the scheduler, we could be doing them already and we're not.

Reply:
I see, thanks for clarifying :bulb: Indeed, this is just a version bump.